linux/opt/sage/bin/rstudio_synapse_cred_env.sh

```diff
@@ -10,7 +10,7 @@
 # In case I am bad at programming
 set -e; set -u; set -o pipefail
 
-AWS_REGION=$(/usr/bin/curl -s http://169.254.169.254/latest/dynamic/instance-identity/document)
+AWS_REGION=$(/usr/bin/curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | /usr/bin/jq -r .region)
```

Comment:
the curl will return JSON so we need to only grab the `"region"` key's value

Reply:
i don't remember, is jq standard on the VM? if not has it been added?

Reply:
It's preinstalled. I was able to use it on my test machine w/out any additional `apt install`s